locales/en.default.schema.json

```diff
@@ -1268,6 +1295,33 @@
           "label": "Desktop image width",
           "info": "Image is automatically optimized for mobile."
         },
+        "image_shape": {
+          "options__1": {
+            "label": "Default"
+          },
+          "options__2": {
+            "label": "Arch"
+          },
+          "options__3": {
+            "label": "Blob"
+          },
+          "options__4": {
+            "label": "Chevron left"
+          },
+          "options__5": {
+            "label": "Chevron right"
+          },
+          "options__6": {
+            "label": "Circle"
```

Comment:
Should probably be "Oval", but if these all have one source of truth this will work itself out.

Reply:
We are using `Circle` for Image with text but `Oval` for product cards because cards would distort the Circle depending on the amount of product grid column and image ratio setting.